composer/core/engine.py

```diff
@@ -127,6 +108,15 @@ def _set_atexit_ran():
 atexit.register(_set_atexit_ran)
 
 
+def _get_default_passes():
+    return [
+        passes.sort_selective_backprop_first,
+        passes.sort_fused_layernorm_last,
+        passes.set_filo_order,
+        passes.warn_if_multiple_loss_interpolation,
+    ]
```

Comment:
Nit: personally would prefer directly importing instead of `passes.X`

Reply:
I normally do as well, but here prefer `passes` since as we add more passes, it's easier to add here without need to to maintain a long list of `from composer.core.passes import x, y, z, .....`